app/models/mixins/event_mixin.rb

```diff
@@ -1,4 +1,11 @@
-# EventMixin expects that event_where_clause is defined in the model.
+# EventMixin expects that event_where_clause_ems_events is defined as most
+# classes have specific columns added to event_streams for their type.
+# Alternatively, a model can override event_where_clause for custom logic.
+#
+# event_where_clause_miq_events defaults to automatically lookup policy events via the
+# belongs_to target (target_type, target_id) association.
+# If this isn't how policy events are created for a class, that class would need to be override
```

Comment:
```suggestion
# If this isn't how policy events are created for a class, that class would need to override
```

Reply:
not sure if override would work here.

We will probably be calling this from reports/table view on the base class so the overridden method won't be called. rigth?

Reply:
Oh good point.  🤷   I'll keep looking.  I originally started this PR because you can't do API requests for events streams of the type "MiqEvent" or even "PolicyEvent" as many places default to returning "EmsEvent" in the results.